src/components/custom-field-input-date/custom-field-input-date.test.jsx

```diff
@@ -159,6 +160,37 @@ describe('src/components/custom-field-input-date/custom-field-input-date', () =>
     });
   });
 
+  describe('calendar icon', () => {
+    it('opens to the correct date on click and closes when clicking away', () => {
```

Comment:
Do we have coverage for this functionality when the user clicks into the text area of the input as well?

Reply:
We have coverage for this now!